_posts/2021-10-03-get-started-with-APIs.md

```diff
@@ -0,0 +1,71 @@
+
+In the simplest of terms, that is what an API does: it acts as the interface between two applications and facilitates information transfer while ensuring speed and security. It is a developer-centric tool - APIs are build by and for developers
+
+## Why to use APIs and their importance
+APIs, or application program interfaces, are vital tools for businesses in all industries. The importance of APIs from a technical standpoint, they allow the capabilities of one computer program to be used by another. They are a means by which two different programs are able to communicate. APIs enable companies to grow their businesses more quickly than ever before and represent an answer for the organizations that are currently spending more than $700 billion per year integrating disparate systems. Like the Web, which opened up the Internet’s potential, APIs are driving a new wave of innovation centered on sharing services. Organizations in all industries are looking to learn more about APIs and their potential to transform business processes.
+
+![](https://media.geeksforgeeks.org/wp-content/uploads/20200517143318/WebAPI.jpg)
+
+The main reason that APIs matter so much in modern markets is that they allow for faster innovation. Barriers to change are reduced and more people can contribute to an organization’s success. They offer two-fold benefits: the company can create better products while standing out from the competition. APIs also make monetization easier. 
+\
+&nbsp;
+## Types of API protocols
+
+
+As the use of web APIs has increased, certain protocols have been developed to provide users with a set of defined rules that specifies the accepted data types and commands. In effect, these API protocols facilitate standardized information exchange:
+
++ SOAP (Simple Object Access Protocol) is an API protocol built with XML, enabling users to send and receive data through SMTP and HTTP. With SOAP APIs, it is easier to share information between apps or software components that are running in different environments or written in different languages.
+
++ XML-RPC is a protocol that relies on a specific format of XML to transfer data, whereas SOAP uses a proprietary XML format. XML-RPC is older than SOAP, but much simpler, and relatively lightweight in that it uses minimum bandwidth.
+
++ REST (Representational State Transfer) is a set of web API architecture principles, which means there are no official standards (unlike those with a protocol). To be a REST API (also known as a RESTful API), the interface must adhere to certain architectural constraints. It’s possible to build RESTful APIs with SOAP protocols, but the two standards are usually viewed as competing specifications.
+
++ JSON-RPC is a protocol similar to XML-RPC, as they are both remote procedure calls (RPCs), but this one uses JSON instead of XML format to transfer data. Both protocols are simple. While calls may contain multiple parameters, they only expect one result.
+
```

Comment:
@itsdharmxd can you just conclude the blog in the end in a decent manner coz after the API protocol, the conclusion is missing so just add up a few lines concluding the blog

Reply:
@Ruchip16  ya sure

Reply:
## Conclusion

The role of APIs is considerably greater if we look at it not only from the software development angle but also from the business collaboration angle. These machine-readable interfaces for resource exchange are like delivery services that work under the hood and enable that needed technological connectivity and make them communicate between the  services.

Reply:
**@Ruchip16**  please review  this

Reply:
after that i will commit

Reply:
LGTM, please create a PR

Reply:
**@LoopThrough-i-j**  ya sure

Reply:
lgtm as well, PR is already been created @itsdharmxd commit your changes and rebase them in this PR itself

> LGTM, please create a PR, after fixing the issues above.